include/osdp.h

```diff
@@ -1049,6 +1049,19 @@ void osdp_cp_teardown(osdp_t *ctx);
 OSDP_EXPORT
 int osdp_cp_send_command(osdp_t *ctx, int pd, const struct osdp_cmd *cmd);
 
+/**
+ * @brief Adds more PD devices in the CP control list.
+ *
+ * @param num_pd Number of PDs connected to this CP. The `osdp_pd_info_t *` is
+ * treated as an array of length num_pd.
+ * @param info Pointer to info struct populated by application.
+ *
+ * @retval 0 on success
+ * @retval -1 on failure
+ */
+OSDP_EXPORT
+int osdp_cp_add_pd(osdp_t *ctx, int num_pd, const osdp_pd_info_t *info);
```

Comment:
Please place this block right after `osdp_cp_setup()` as they are part of the lifecycle management methods.

Reply:
done